src/main/java/teammates/common/datatransfer/attributes/StudentAttributes.java

```diff
@@ -158,6 +158,14 @@ public String getComments() {
         return comments;
     }
 
+    public void setGoogleId(String googleId) {
+        this.googleId = googleId;
+    }
+
+    public void setComments(String comments) {
```

Comment:
Just a suggestion, maybe the method can return `this` instead to allow for method chaining (i.e. `s.setGoogleId(null).setComments(null)`). Not a big issue though

Reply:
Interesting point, @hcwong what do you think?

Reply:
I think we can actually delete these 2 setter functions